styles.css

```diff
@@ -493,3 +493,301 @@ textarea {
 .contact-form-end-image {
   margin-left: 100px;
 }
+
+.desktop-navigation {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    width: 100%;
+  }
```

Comment:
- [ Optional ] As suggested by the previous reviewer Kindly I recommend you it would be better to use the same on the mobile version to make it responsive until the desktop screen size.